pytorch_lightning/trainer/connectors/slurm_connector.py

```diff
@@ -1,72 +1,15 @@
 import os
-import re
 import signal
 from subprocess import call
 
-import torch
-import torch.distributed as torch_distrib
-
 from pytorch_lightning import _logger as log
-from pytorch_lightning.utilities import DeviceType, DistributedType
-from pytorch_lightning.utilities.distributed import rank_zero_info
 
 
 class SLURMConnector:
 
     def __init__(self, trainer):
         self.trainer = trainer
 
-    def on_trainer_init(self, num_gpu_nodes):
```

Comment:
Where did that go ?

Reply:
it's not needed anymore the constructor is enough, some logic has moved to the slurmenvironment plugin.